lib/tasks/administrator.rake

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+namespace :admin do
+  task create: :environment do
+    # Create administrator account if it doesn't exist
+    unless User.where(email: "admin@admin.com").exists?
+      admin = User.create(name: "Administrator", email: "admin@admin.com", password: "administrator",
```

Comment:
The task should receive parameters to allow system administrators to create their own account if they want. If non is provided then we use the example as default